src/middlewared/middlewared/plugins/ntp.py

```diff
@@ -100,6 +100,8 @@ class Config:
         datastore_prefix = 'ntp_'
         cli_namespace = 'system.ntp_server'
         entry = NTPServerEntry
+        prefix = 'SYSTEM_ADVANCED'
+        role_prefix = 'NETWORK_INTERFACE'
```

Comment:
There may be a **very** good reason for this, but I'm wondering why `NETWORK_GENERAL` wouldn't be a better fit?